### PR TITLE
Tom add microservice backend

### PR DESCRIPTION
**Pull request by tladendo**
yo @indytechcook @vietnogi check it out

### CODE REVIEW

File: src/client/app/index.jade [hunk 1]
```diff
@@ -1,6 +1,6 @@
 - var scripts = []
-- if (stubs) scripts.push('/src/client/app/project/project.stub')
-- if (stubs) scripts.push('/bower_components/angular-mocks/angular-mocks')
+- scripts.push('/bower_components/angular-mocks/angular-mocks')
+- scripts.push('/src/client/app/project/project.stub')
```
Comment (vietnogi) [inline src/client/app/index.jade]: Will this get included to our production code?

File: src/client/app/project/project.service.js [hunk 2]
```diff
@@ -55,7 +52,13 @@
 
       function getProjectComplete(data, status, headers, config) {
         logger.info('project data', data);
-        return data.data.result.content;
+        var project = data.data.result.content;
+        project.styleMinimalComplex = parseInt(project.styleMinimalComplex)
```
Comment (vietnogi) [inline src/client/app/project/project.service.js]: loop?

File: src/client/app/index.jade (hunk 1 above)
Comment (indytechcook) [inline src/client/app/index.jade]: I echo this concern

**Comment by indytechcook** [comment]
other then the stubs check, looks good.

File: gulpfile.js [hunk 3]
```diff
@@ -74,7 +74,7 @@ gulp.task('scss', function () {
 gulp.task('jade', function () {
   log('Compiling Jade --> HTML');
 
-  var stubs = config.useStubs ? true : false;
+  var stubs = config.useStubs === 'true';
```
Comment (vietnogi) [inline gulpfile.js]: Is there a way to handle stubs without intruding with jade files?  Don't want to intertwine the two layers and have to decouple them later.

**Comment by tladendo** [comment]
@indytechcook @vietnogi -- I updated so that stubs are only pulled in when we ask for them. Else we pull in "container" versions of the stubs that do nothing. I investigated other solutions, and unfortunately, I don't think any of them are particularly elegant. This seems manageable, at least, to me :)

**Comment by vietnogi** [comment]
LGTM if you create a tech dept lira to decouple the stubbing solution to our app code.

**Pull request merged**